examples/config files - basic/3 connector-ldap.yml

```diff
@@ -17,11 +17,22 @@
 # You must specify all four of these settings.  Consult with your
 # enterprise directory administrators to get suitable values.
 # These access credentials are sensitive and must be protected.
-username: "LDAP username goes here"
+username: "LDAP or Credential Manager username goes here"
```

Comment:
Doesn't this have to be the LDAP user name?  You then have to use that same name in the credential manager.  If it is the CM user name, where does the LDAP username come from?